Onboarding/Onboarding/Pages/ProfilePages/Language.cs

```diff
@@ -12,121 +12,125 @@ namespace Onboarding.Pages.ProfilePages
 {
     public class Language
     {
-        public void AddALanguage(IWebDriver driver, string language, string languagelevel)
+        public IWebDriver driver;
+        Language LanguageObj;
+        public Language(IWebDriver _driver)
+        { 
+            this.driver = _driver;
+        }
+        public void LanguageStepDefinitions()
```

Comment:
This I believe is not needed is there any reason you are doing this?

Reply:
redundant